FINESSE_GUI/serial/serial_view.py

```diff
@@ -0,0 +1,145 @@
+"""Panel and widgets related to the control of the serial ports."""
+from copy import deepcopy
+from functools import partial
+from typing import Dict, Sequence
+
+from PySide6.QtWidgets import QComboBox, QGridLayout, QLabel, QPushButton, QWidget
+
+
+class SerialPortControl(QWidget):
+    """Widgets to control the communication with a single serial port."""
+
+    def __init__(
+        self,
+        devices: Dict[str, Dict[str, str]],
+        avail_ports: Sequence[str],
+        avail_baud_rates: Sequence[str],
+    ) -> None:
```

Comment:
Would you mind adding a docstring here?

I'd like to enable warnings for missing docstrings (#42) so that we can have total coverage before tackling the MkDocs stuff.